From 76321bb4f7ee9c6edd1fe4a4dafe3b7abf0fa4d7 Mon Sep 17 00:00:00 2001
From: Alexey Donoga <adonoga@determine.com>
Date: Fri, 15 Jul 2016 08:59:15 +0300
Subject: [PATCH] Fix for issue #492 (#496)

Murrine engine update causes misalignment of icon text and shadow. Fix by @makitso

Bug: https://github.com/numixproject/numix-gtk-theme/issues/492
Bug-Ubuntu: https://launchpad.net/bugs/1598316
Origin: https://github.com/numixproject/numix-gtk-theme/commit/76321bb4f
---
 gtk-2.0/gtkrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk-2.0/gtkrc b/gtk-2.0/gtkrc
index 7157c41..9ca4e8d 100755
--- a/gtk-2.0/gtkrc
+++ b/gtk-2.0/gtkrc
@@ -528,7 +528,7 @@ style "xfdesktop-icon-view" {
 	fg[ACTIVE] = @selected_fg_color
 
 	engine "murrine" {
-		textstyle = 5
+		textstyle = 0
 		text_shade = 0.05
 	}
 }
-- 
2.8.1

